sc-cli.go

```diff
@@ -0,0 +1,9 @@
+package main
```

Comment:
I can, I was just following their recommended set up.
https://github.com/spf13/cobra#getting-started

Perhaps, we could have
cli/
 sc-cli.go
 Makefile
 cmd/
    commands then go here.